app/components/member/Teams/chooser.js

```diff
@@ -168,7 +168,7 @@ class Chooser extends React.Component {
 
 export default Relay.createContainer(Chooser, {
   initialVariables: {
-    showingDialog: false,
+    isMounted: false,
```

Comment:
Oh you're totally right, I was rushing through some last minute changes and this is poorly named. I'll change them all back to `showingDialog` which is actually what it _is_